Gravity/Gravity/Gravity.csproj

```diff
@@ -76,7 +76,7 @@
     <Compile Include="DAL\RSAPI\RsapiDao.Update.cs" />
     <Compile Include="Extensions\RdoExtensions.cs" />
     <Compile Include="Extensions\TypeExtensions.cs" />
-    <Compile Include="Globals\SharedConstants.cs" />
+    <Compile Include="obj\SharedConstants.cs" />
```

Comment:
Getting error Source file could not be found.

Reply:
Fixed in newest push